configs/components/puppet.rb

```diff
@@ -60,7 +60,79 @@
   when "aix"
     pkg.install_service "resources/aix/puppet.service", nil, "puppet"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [
+      # Seriously?... http://stackoverflow.com/questions/9419411/wix-setting-service-startup-type-using-a-property-property-not-recognized
+      # Yep, Seriously - http://sourceforge.net/mailarchive/forum.php?thread_name=CANJN1a6gDE2tUUnP4SE05m8Ojh2q0Y8LrWOxbrF89YyJJCHn1A%40mail.gmail.com&forum_name=wix-users
+      {
+        :directory_ref => "bindir",
+        :id => "PuppetServiceAutomatic",
+        :guid => "639ECD7F-6186-43D5-9E1A-FC0278DBEE15",
+        :win64 => true,
+        :conditions => ['<![CDATA[ (PUPPET_AGENT_STARTUP_MODE ~<> "manual") AND (PUPPET_AGENT_STARTUP_MODE ~<> "disabled") ]]>'],
+        :file => {
+          :id => "RubyExeAutomatic",
+          :source => 'stagedir\\sys\\ruby\\bin\\ruby.exe',
+        },
+        :serviceinstall => {
+          :id => "ServiceInstaller",
+          :description => "Periodically fetches and applies configurations from a Puppet master server.",
```

Comment:
A suggestion - since we are using the same content here (titles etc), we should really but them up in comment variables, e.g. service_description, display_name etc. etc.
That way, even though we have some of the horribleness of repetition, we can protect against mis-types in the service descriptions etc. etc.